command/ripe_atlas/credits.go

```diff
@@ -0,0 +1,72 @@
+package ripe_atlas
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/innogames/slack-bot/v2/bot"
+	"github.com/innogames/slack-bot/v2/bot/matcher"
+	"github.com/innogames/slack-bot/v2/bot/msg"
+	"github.com/innogames/slack-bot/v2/client"
+	"github.com/pkg/errors"
```

Comment:
Try to avoid the `pkg/errors` package in your code. Go has sufficient capabilities to work with errors nowadays. For example you can create errors with `fmt.Errorf()`. Additionally if you use the `%w` placeholder to encapsulate another error, that is effectively a `errors.Wrap()` call.
If you want to familiarize with additional error handling logic, check the `errors` package from Go's stdlib instead

Reply:
TIL. I removed all pkg/errors usage.